security/s2a/internal/crypter/common.go

```diff
@@ -0,0 +1,15 @@
+package crypter
+
+// S2AAeadCrypter is the interface for an AEAD cipher used by the S2A record protocol.
+type S2AAeadCrypter interface {
+	// Encrypt encrypts the plaintext and computes the tag of dst
```

Comment:
Please try to fit as much as possible in 80 chars. For example, some of the comment text on the next line can be moved to the end of this line.

Same below.